dynamics/src/include/BrittleCGDynamicsKernel.hpp

```diff
@@ -157,14 +168,11 @@ template <int DGadvection> class BrittleCGDynamicsKernel : public CGDynamicsKern
     // Common brittle parts of the momentum solver.
     void updateMomentum(const TimestepTime& tst) override
     {
-        static const double cosOceanAngle = cos(radians * params.ocean_turning_angle);
-        static const double sinOceanAngle = sin(radians * params.ocean_turning_angle);
-
 #pragma omp parallel for
         for (size_t i = 0; i < u.rows(); ++i) {
             // FIXME dte_over_mass should include snow in the total mass
             const double dteOverMass = deltaT / (params.rho_ice * cgH(i));
-            // Memoized initial velocity values
+            // Memorized initial velocity values
```

Comment:
No, this is definitely 'memoized'

Reply:
Memoized it is - and a new word in my personal dictionary too!